communication/slack-config/sig-release/usergroups.yaml

```diff
@@ -48,36 +50,24 @@ usergroups:
       - release-notes
       - sig-release
     members:
-      - cpanato # SIG Release Technical Lead
-      - jeremyrickard # SIG Release Chair
-      - justaugustus # SIG Release Chair
-      - mehabhalodiya # 1.29 RT Lead Shadow
-      - mickeyboxell # 1.29 RT Lead Shadow
-      - neoaggelos # 1.29 RT Lead Shadow
-      - Priyankasaggu11929 # 1.29 RT Lead
-      - puerco # SIG Release Technical Lead
-      - ramrodo # 1.29 RT Lead Shadow
-      - salaxander # 1.29 Emeritus Adviser
-      - saschagrunert # SIG Release Chair
-      - Verolop # SIG Release Technical Lead
-
-  # Should match SIG Release Leads at all times:
-  # https://git.k8s.io/community/sig-release/README.md#leadership
-  - name: sig-release-leads
-    long_name: SIG Release Leads
-    description: >-
```

Comment:
@fsmunoz, this section (`sig-release-leads` user group) needs to be re-added.

Reply:
Done.